coordinator/src/orderbook/websocket.rs

```diff
@@ -18,9 +24,66 @@ use std::sync::Arc;
 use std::time::Duration;
 use tokio::sync::broadcast::error::RecvError;
 use tokio::sync::mpsc;
+use tokio::task::spawn_blocking;
+use uuid::Uuid;
 
 const WEBSOCKET_SEND_TIMEOUT: Duration = Duration::from_secs(5);
 
+async fn handle_insert_order(
+    state: Arc<AppState>,
+    trader_id: PublicKey,
+    order: NewLimitOrder,
+) -> Result<()> {
+    if order.trader_id != trader_id {
+        bail!("Maker {trader_id} tried to trade on behalf of someone else: {order:?}");
+    }
+
+    tracing::trace!(?order, "Inserting order");
+
+    let order = spawn_blocking({
+        let mut conn = state.pool.clone().get()?;
+        move || {
+            let order = orders::insert_limit_order(&mut conn, order, OrderReason::Manual)?;
+
+            anyhow::Ok(order)
+        }
+    })
+    .await??;
+
+    let _ = state
+        .trading_sender
+        .send(NewOrderMessage {
+            order,
+            channel_opening_params: None,
+            order_reason: OrderReason::Manual,
+        })
+        .await;
+
+    Ok(())
+}
+
+async fn handle_delete_order(
+    state: Arc<AppState>,
+    trader_id: PublicKey,
+    order_id: Uuid,
+) -> Result<()> {
+    tracing::trace!(%order_id, "Deleting order");
+
+    spawn_blocking({
+        let mut conn = state.pool.clone().get()?;
+        move || {
+            orders::delete_trader_order(&mut conn, order_id, trader_id)?;
+
+            anyhow::Ok(())
+        }
+    })
+    .await??;
+
+    let _ = state.tx_orderbook_feed.send(Message::DeleteOrder(order_id));
```

Comment:
`order_id` is the `trader_order_id` right? If so, the traders receiving the `DeleteOrder` msg might not know what todo with this.

The user connected to the websocket shouldn't know about the `trader_order_id`, so did this ever work? 🙈

Reply:
Maybe I made a mistake. I'll test it again locally.

Reply:
No, you didn't it's been like that before.

Reply:
Hmm, I've been thinking about it and I don't see the problem with those warnings we get on the app.

As far as I can tell, what happens is:

1. The maker generates an order.
2. The maker deletes the order.
3. The app starts and connects to the coordinator's WS, getting an `AllOrders` message that does not include the deleted order.
4. The WS still sends the app the corresponding `DeleteOrder` message from step 2.

I don't understand why the "outdated" message is delivered to the app, but I think we should demote the WARN to DEBUG because I think this is harmless.

Reply:
I looked into it and we are actually sending the `trader_order_id` to all users 🙈 It's probably not an issue, but I think it wasn't meant to be used like that: 

Here are we mapping the `trader_order_id` from the db to `order.id` for the orders we are returning to the users.

https://github.com/get10101/10101/blob/7c6f8ee0a4fc1574f7d4c8000da4b8bfa1b6012a/coordinator/src/orderbook/db/orders.rs#L114

Reply:
> I don't understand why the "outdated" message is delivered to the app, but I think we should demote the WARN to DEBUG because I think this is harmless.

Imho this happens because we are here also clinsing outdated orders [here](https://github.com/get10101/10101/blob/a862e02415deeba27f194539f495dba4c02c2224/coordinator/src/orderbook/trading.rs#L160-L166) and we might have a race condition where the maker deletes an order and this is run

Reply:
> I looked into it and we are actually sending the `trader_order_id` to all users 🙈 It's probably not an issue, but I think it wasn't meant to be used like that:
> 
> Here are we mapping the `trader_order_id` from the db to `order.id` for the orders we are returning to the users.
> 
> https://github.com/get10101/10101/blob/7c6f8ee0a4fc1574f7d4c8000da4b8bfa1b6012a/coordinator/src/orderbook/db/orders.rs#L114

Yeah, that makes sense.

Did we ever have an _order_ ID that wasn't the trader order ID though? I can't find another one.

Reply:
Yes, on DB level we have a different one.

Reply:
Okay, but that one has only ever been used internally:

```rust
    // this id is only internally but needs to be here or diesel complains
    #[allow(dead_code)]
    pub id: i32,
```